app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js

```diff
@@ -80,9 +80,22 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
       if (!thisel.querySelectorAll('.js-toggle-link').length) {
         var span = document.createElement('span')
-        span.className = 'gem-c-step-nav__toggle-link js-toggle-link'
-        span.setAttribute('aria-hidden', true)
-        span.setAttribute('hidden', 'hidden')
+        var showHideSpan = document.createElement('span')
+        var commaSpan = document.createElement('span')
+        var thisSectionSpan = document.createElement('span')
+
+        span.classList.add('gem-c-step-nav__toggle-link')
+        showHideSpan.classList.add('js-toggle-link')
+        commaSpan.classList.add('govuk-visually-hidden')
+        thisSectionSpan.classList.add('govuk-visually-hidden')
+
+        commaSpan.innerHTML = ', '
```

Comment:
Voiceover pronounces this comma, which is a bit odd (it doesn't do it for commas in a normal sentence). I think it's because it's in a span by itself, which isn't something we can get around. I tried other punctuation but it pronounces that too. Raising as something to ask Anika about.

Reply:
I'm not able to replicate this locally with voiceover but this might be a version thing, which obviously isn't an excuse. Will raise with Anika as you say.

Reply:
A way around this could be to add an `aria-label` attribute with what you'd like the screenreader to say. Testing that on VoiceOver on Safari it read things out without reading the comma.

Reply:
@injms Are you thinking an empty span with an `aria-label`? Or an `aria-label` on the original span? I haven't tested this yet so will muck around.

Reply:
I _think_ adding it to the original span, so then the visually hidden classes wouldn't be needed.

For example: 

```html
<span aria-label="Show this section">Show</span>
```

Reply:
From Anika:
>you mean an aria-label specifically for "show this section"? I don't know if that would fix it. It might be worth a try. Although, that would also mean Google will see "Do a thing, show" instead of "Do a thing, show this section". And I think the latter would be better for the Google case?

I played around and came up with this:

```
<span>
<span class="govuk-visually-hidden" aria-label=", ">, </span>
<span class="gem-c-step-nav__toggle-link js-toggle-link">show</span>
<span class="govuk-visually-hidden" aria-label=" this section"> this section</span>
</span>
```

I don't know if this is overkill but it works on my local. I don't know if this is an anti-pattern or not, but I wonder if this solves Andy's problem and maybe the NVDA problem that Anika raised on the card. If we think this is a bad idea, based on Anika's comments on the card, I reckon we should press on with the current setup as, to quote Anika, it's better than it was.